services/printers/paperboy/pyproject.toml

```diff
@@ -1,18 +1,15 @@
-[tool.poetry]
+[project]
```

Comment:
Missing `[build-system]` section (IIRC this is needed for it to build properly). `setuptools` seems pretty standard for this purpose.